releasenotes/notes/evolution-framework-primitives-c86779b5d0dffd25.yaml

```diff
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+    Added `~qiskit.algorithms.time_evolvers` package with interfaces that will cover
+    primitive-enabled time evolution algorithms:
+    :class:`~qiskit.algorithms.time_evolvers.EvolutionProblem`,
+    :class:`~qiskit.algorithms.time_evolvers.EvolutionResult`,
+    :class:`~qiskit.algorithms.time_evolvers.ImaginaryEvolver`,
+    :class:`~qiskit.algorithms.time_evolvers.RealEvolver`.
+deprecations:
+  - |
+    `~qiskit.algorithms.evolvers` package will now issue a ``PendingDeprecationWarning``.
```

Comment:
I think we should also have more along the lines of text we are using elsewhere that additionally says along the lines of "This package will be deprecated in a future release and subsequently removed after that. This is being replaced by the new  `~qiskit.algorithms.time_evolvers` package."